src/main/java/com/infoshareacademy/entity/product/ProductRecipe.java

```diff
@@ -29,65 +35,13 @@ public class ProductRecipe extends Product {
     @JoinColumn(name = "recipe_id")
     private Recipe recipe;
 
-    public ProductRecipe() {
-    }
-
-    public ProductRecipe(String productName, Double amount) {
-        this.productName = productName;
-        this.amount = amount;
-    }
-
-    public ProductRecipe(String productName, Double amount, ProductUnit unit) {
-        this.productName = productName;
-        this.amount = amount;
-        this.unit = unit;
-    }
-
-    public void setProductId(Long id) {
-        this.productId = id;
-    }
-
-    public Long getProductId() {
-        return productId;
-    }
-
-    public Recipe getRecipe() {
-        return recipe;
-    }
-
-    public void setRecipe(Recipe recipe) {
-        this.recipe = recipe;
-    }
-
-    public String getProductName() {
-        return productName;
-    }
-
-    public void setProductName(String productName) {
-        this.productName = productName;
-    }
-
-    public Double getAmount() {
-        return amount;
-    }
-
-    public void setAmount(Double quantity) {
-        this.amount = quantity;
-    }
-
-    public ProductUnit getUnit() {
-        return unit;
-    }
-
-    public void setUnit(ProductUnit unit) {
-        this.unit = unit;
-    }
 
     @Override
     public String toString() {
 
         return  productName + '\'' +
-                ", ilość:" + amount + " " + unit.getValue();    }
+                ", ilość:" + amount + " " + unit.getValue();
+    }
```

Comment:
Jak używacie lomboka z adnotacją @Data to metoda toString nie jest wam potrzebna, no chyba, że chcecie ją jakoś wyjątkowo zmienić